### PR TITLE
Eng 7576 vmc semicolons

### PR DESCRIPTION
**Pull request by paulmartel**
Successfully tested by hand in sqlcmd and VMC.
Was unable to verify if my added VMC geb test statements work as expected -- they MIGHT be badly broken. We'll know when the geb framework is back on its feet?

### CODE REVIEW

File: src/frontend/org/voltdb/dbmonitor/js/voltdb.queryrender.js [hunk 1]
```diff
@@ -360,7 +243,15 @@ function QueryUI(queryString, userName) {
...
 
         } else {
-            target.append('<span class="errorValue">Error: ' + response.statusstring + '\r\n</span>');
+            // This inline encoder hack is intended to use html's &#nnnn; character encoding to
+            // properly escape characters that would otherwise mean something as html
+            // -- including angle brackets and such that are commonly used to suggest that the
+            // user correct their ddl grammar. Angle-bracketed place-holders were being
+            // rendered as invisible meaningless html tags.
+            // See http://stackoverflow.com/questions/18749591/encode-html-entities-in-javascript#18750001
+            var encodedStatus = response.statusstring.replace(/[\u00A0-\u9999<>\&]/gim,
```
Comment (akhanzode) [inline src/frontend/org/voltdb/dbmonitor/js/voltdb.queryrender.js]: Does this fix ENG-7575?

**Comment by jxchinaren** [comment]
@paulmartel The clean up looks generally good to me. Do you need to create another GEB testing ticket for your change? Maybe ask John to verify later?

**Comment by jxchinaren** [comment]
@paulmartel I am fine to merge this as you have done manual tests. ENG-8183 created for GEB testing for new added tests in this ticket.

File: src/frontend/org/voltdb/dbmonitor/js/voltdb.queryrender.js (hunk 1 above)
Comment (paulmartel) [inline src/frontend/org/voltdb/dbmonitor/js/voltdb.queryrender.js]: Yes. ENG-7575 can be closed when this merges.

**Pull request merged**